src/run.py

```diff
@@ -23,8 +23,12 @@
 from apache_beam.options.pipeline_options import PipelineOptions, SetupOptions
 from apache_beam.runners.runner import PipelineResult
 
-from .config import ModelConfig, SinkConfig, SourceConfig
-from .pipeline import build_pipeline
+try:
+    from .config import ModelConfig, SinkConfig, SourceConfig
+    from .pipeline import build_pipeline
+except ImportError:
+    from config import ModelConfig, SinkConfig, SourceConfig
+    from pipeline import build_pipeline
```

Comment:
No, we cannot. The issue is the difference between python -m src.run (used by the local run) and python src/run.py (used by the template launcher).

Reply:
there gotta be a way to avoid this branching.

Reply:
we can just ask users to always do `python src/run.py` not using `-m` or we can install it as the package (this is not popular if your code is not shared across different Python projects). Anyway, the example here is more like the code sample for users to take. This is just one way to avoid the import error.

Reply:
please use https://github.com/google/dataflow-ml-starter/pull/19 for more discussions.